website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+entity and so be impacted by the same policies. This is restoring the intended
+Vault behavior and is the only behavior possible without the bugs that allowed
+the duplicates to be created.
+
+#### Case 2: intentional different-case duplicates
+
+If the different-case variations of an alias name actually represent different
+logical users or entities in an external system then activating the flag will
+merge them and potentially grant users access to resources they shouldn't have.
+It is critical to ensure these are resolved before activating the flag. Use the
+steps below to choose an resolution for each duplicate name on each mount.
+
+<Note> Using case-sensitive names is unsupported and carries significant
+  security risks. There is a risk that different-case alias duplicates could
+  merge at any time due to unrelated changes in storage such as other entities
+  or groups being deleted. You should aim to resolve these duplicates as soon as
+  possible to restore Vault's intended security model. </Note>
+
+ 1. If you can confirm that the duplicate aliases or their entire auth mounts
+    are unused or unimportant, they can be deleted by their ID as shown in the
+    log line, for example `id=df3568a4-3b65-4104-9481-1129ecbed72f`. Use the
+    [entity alias delete
+    endpoint](https://developer.hashicorp.com/vault/api-docs/secret/identity/entity-alias#delete-entity-alias-by-id).
+    Deleting all but one of each duplicate name on each mount is sufficient to
+    avoid unintentional merging when the flag is activated.
+ 2. If you are unsure whether these aliases are used, you will need to discover
+    this and form a strategy for resolving.
+
+    You may need to reconfigure the auth mount or external service to ensure
+       that it will no longer create aliases with names that differ only-by
+       case. For example by including a unique ID in the alias name as well as
+       the username. There might be some disruption to users as you roll out
+       that change but it will depend on the chosen solution.
+
+    We anticipate this being rare and are not aware of any specific example
+    integrations where this would apply. Vault Enterprise customers should work
+    with HashiCorp support staff to determine the best strategy if you find this
+    is the case.
+
+    Once you are able to manually resolve the duplicates via deletion or
+    reconfiguration and no longer see them reported during unseal you are safe
+    to activate the flag.
+
+
+### Resolving _local_ different-case entity alias duplicates
+
+Local different-case entity alias duplicates have all the same considerations as
+[non-local entity alias
+duplicates](#resolving-different-case-entity-alias-duplicates).
+
+The only difference is that these aliases only exist on the local cluster. If
+the cluster that reports them is a Performance Replication secondary, you will
+need to perform any resolution steps necessary against that cluster and not the
+Primary.
+
```

Comment:
What happens to same-case entity alias duplicates? I don't see a section for that. Is it so simple that it can be covered right away before jumping into all these different-case scenarios?

Reply:
I think in Aarah's restructure this isn't exactly dealt with but is less of an issue as it's more split up into specific trouble shooting guides. It still seems cumbersome to find a place to explain why not other types so I think her structure is sufficient to guide the user through without the extra background.